public_html/storefront/view/default/template/embed/checkout/confirm.tpl

```diff
@@ -81,8 +82,9 @@
 			<?php if ($payment_method) { ?>
 				<tr>
 					<td class="align_left"><?php echo $payment_firstname . ' ' . $payment_lastname; ?>
-						<br/><?php echo $telephone; ?></td>
-			<?php echo $this->getHookVar('checkout_confirm_payment_info_1'); ?>
+						<br/><?php echo $telephone; ?>
+						<?php echo $this->getHookVar('checkout_confirm_payment_info_1'); ?>
```

Comment:
We pasted hookvar here for adding new column. 
In your variant you changing column content. 
Why do you do that way? Why not via updateData()  (i mean changing variable $telephone inside aview) ?

Reply:
We need to add extra content under the telephone and within the same td wrapper instead of outside the telephone wrapper.

Reply:
You don't need to move hookvar inside TD tag for that. 
Why do you do not changing $telephone variable via $that->view->assign('telephone', $telephone + $your_extra_html_data ); ? You can append some data to telephone via UpdatData method.
i'm afraid your changes int this place affects on other ready extensions that uses this hookVar too.

Reply:
I will try your suggestion first and see the result.

Reply:
I have followed your suggestion and it works. And i have changed the code.